packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.tsx

```diff
@@ -6,6 +6,8 @@ import {
 import { IResizeGroupProps } from './ResizeGroup.Props';
 import styles = require('./ResizeGroup.scss');
 
+const RESIZE_DELAY = 16;
```

Comment:
100ms is pretty noticeable, the top gif is 100ms, as you can see it doesn't keep up with the user resizing the window. 
![100msdelay](https://cloud.githubusercontent.com/assets/1581488/26465575/9fa497e6-4140-11e7-8493-89403cb8a798.gif)
![16msdelay](https://cloud.githubusercontent.com/assets/1581488/26465574/9fa46d7a-4140-11e7-9a72-ea29d9085aaa.gif)